nss-day-cohort-48/giffygram-jumpstation
# Login and registering

#### Changes Made
   changed API port to correct local host
   Added login, logout, and registering functionality to the application. 


#### Steps to Review

1. Checkout this branch locally.
   ```
git pull origin login-and-registering
on local host create an account, login with account, and then logout. Repeat process to ensure no bugs
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > on local host create an account, login with account, and then logout. Repeat process to ensure no bugs
   > when user creates account they are directed back to login form. once logged in they are directed to the main application. If new user, they will click create account and start a new account. If user enters in a previously used email it will window alert them that it is taken and stay on the register for 

4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
